charts/ldap-scim-bridge/values.yaml

```diff
@@ -0,0 +1,42 @@
+image:
+  repository: quay.io/wire/ldap-scim-bridge
```

Comment:
How is this being built?

docs?  Can I have something like "read documentation *here* on how to get a test instance running; then run *these* commands to start the ldap-scim-bridge tests."

Reply:
See https://github.com/wireapp/wire-server/pull/1709/files#diff-14eedc550a0f3ad7f0f359b4fa085bb2584d09d0e37b0749e8e58ade876b3f21

Reply:
> How is this being built?

Built and push manually until I have a working version.

Reply:
Done in https://github.com/wireapp/ldap-scim-bridge/pull/8